### ISSUE TITLE
GKE Github Actions guide does not work as written, missing Kustomize info

### ISSUE THREAD

**Issue by AlexHolderDeveloper** [google-github-actions/setup-gcloud#269]
### TL;DR
The guide for using Github Actions to deploy Docker images to GKE has an example workflow that uses Kustomize, but the guide itself makes no mention of Kustomize to the point where you can NOT use the example workflow & expect it to work if you follow the guide to deploy a repository other than the example static site project. 

**Expected behavior**
After following [this guide](https://github.com/google-github-actions/setup-gcloud/blob/master/example-workflows/gke/README.md) to the letter and using my own created repository (which the guide says I can do!), I expect the example workflow to be able to deploy new images of my repository to GKE.

**Observed behavior**
When the workflow reaches the "Deploy" step and runs this command:
`Run ./kustomize edit set image gcr.io/PROJECT_ID/IMAGE:TAG=gcr.io/$PROJECT_ID/$IMAGE:$GITHUB_SHA`
The step of the example workflow errors out, with its brief error being:
`Error: Process completed with exit code 1.`
And its logged error being:
`Error: Missing kustomization file 'kustomization.yaml'.`

### Reproduction
Follow this guide and use a new Github repository instead of the example repository.
[https://github.com/google-github-actions/setup-gcloud/blob/master/example-workflows/gke/README.md](https://github.com/google-github-actions/setup-gcloud/blob/master/example-workflows/gke/README.md)

The app you use in the repo shouldn't matter, as long as it runs as a Docker image/container when you try to do that outside of this deployment guide. I have an ExpressJS server that gets bundled into a Docker image, and I can run that perfectly fine as both a local NodeJS app and as a locally-run Docker container. 

### Proposed Solution
Make it clear that the Kustomize-related portions of the example workflow require these three files in the root of the repository to make the workflow run as the guide intends:

- [kustomization.yml](https://github.com/google-github-actions/setup-gcloud/blob/master/example-workflows/gke/kustomization.yml) 
- [deployment.yml](https://github.com/google-github-actions/setup-gcloud/blob/master/example-workflows/gke/deployment.yml)
- [service.yml](https://github.com/google-github-actions/setup-gcloud/blob/master/example-workflows/gke/service.yml)

Once I added those three files to the root of my repository (& modified values to have relevant app names & ports for my project), everything works perfectly.

Would be nice to also have those things explained -- what is Kustomize? What is it doing? Why can't gcloud or other things handle that functionality? 
As a complete newb to GKE, I have zero context for what is happening here. I know that guides that don't explain anything beyond "put this value here to make this work" have their place, but even this guide has gaps when it comes to an apparently-crucial part of the example.

**Comment by FrodoTheTrue** [comment]
Hey @AlexHolderDeveloper !
We have updated our GKE examples. There are two of them for now: simple and kustomize versions.

1) If you simply want to deploy hello-world app, you can use `example-workflows/gke` folder
2) If you want more production way, where you will build your own app and docker image, you can use `example-workflows/gke-kustomize` folder

**Comment by AlexHolderDeveloper** [comment]
Thanks, @FrodoTheTrue . 

I don't have capacity to go back through this stuff & verify things as I've moved my work across to Cloud Run, but at a glance the updates seem very helpful. 

Happy to close & move on if that's okay?
